### PR TITLE
Fix markdown for a link

### PR DESCRIPTION
**Pull request by andynava**
## Summary

Since the link's URL includes a close parentheses, you need to add another one so markdown will render the link correctly.

### CODE REVIEW

File: docs/architecture/cloud-native/azure-data-storage.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ You manage data in Cosmos DB data by creating [databases, containers, and items]
...
 
-Note in Figure 5-17 how you start by creating a Cosmos DB database inside of an Azure account. That database becomes the unit of management for a set of containers. A container is a schema-agnostic grouping of items that can be expressed as a collection, table, or graph, based on your selected API provider (discussed in the prior section). Items are the data that you add to the container and are represented as documents, rows, nodes, or edges. By default, all items that you add to a container are automatically indexed without requiring explicit index or schema management.
+Note in Figure 5-17 how you start by creating a Cosmos DB database inside of a database account. That database becomes the unit of management for a set of containers. A container is a schema-agnostic grouping of items that can be expressed as a collection, table, or graph, based on your selected API provider (discussed in the prior section). Items are the data that you add to the container and are represented as documents, rows, nodes, or edges. By default, all items that you add to a container are automatically indexed without requiring explicit index or schema management.
```
Comment (andresnava) [inline docs/architecture/cloud-native/azure-data-storage.md]: The image shows the databases under "Database Accounts", not "Azure account"

**Comment by mairaw** [comment]
@ardalis @nishanil please review

**Comment by mairaw** [comment]
@robvet

File: docs/architecture/cloud-native/azure-data-storage.md (hunk 1 above)
Comment (robvet) [reply docs/architecture/cloud-native/azure-data-storage.md]: Great catch! Agree with the change. But, I will add it in my upcoming Pull Request for this chapter. That way, we only do a single merge.

**Review by mairaw — APPROVED** [review]
Thanks @andresnava. Synced with @robvet and this one is good to merge.

**Pull request merged**